tools/pdc2png/wscript

```diff
@@ -50,9 +50,11 @@ def build(bld):
     pdc2png_env.CFLAGS.append('-I' + bld.path.abspath() + '/../src/fw/util/time')
     pdc2png_env.CFLAGS.append('-I' + bld.path.abspath() + '/src/include')
 
+    pdc2png_env.CFLAGS.append('-Wno-gnu-variable-sized-type-not-at-end')
+
     # time_t is defined in sys/types in newlib, and time.h on recent Linux
     # so just force the defined type for testing time
-    pdc2png_env.CFLAGS.append('-Dtime_t=__SYSCALL_SLONG_TYPE')
+    #pdc2png_env.CFLAGS.append('-Dtime_t=__SYSCALL_SLONG_TYPE')
```

Comment:
Can this just be removed (I think this was commented before, was the change maybe just lost)?

Reply:
Removed. This is a separate define from the last one. Forgot about it during the last fix.